Trappist/src/Promact.Trappist.Web/wwwroot/app/conduct/register/register.component.ts

```diff
@@ -1,8 +1,42 @@
 ﻿import { Component } from '@angular/core';
+import { TestAttendees } from './register.model';
+import { ConductService } from '../conduct.service';
+import { Router } from '@angular/router';
 
 @Component({
     moduleId: module.id,
     selector: 'register',
     templateUrl: 'register.html',
 })
-export class RegisterComponent { }
+export class RegisterComponent {
+    testAttendees: TestAttendees = new TestAttendees();
+    isErrorMessage: boolean;
+    testId: string;
+    registrationUrl: string;
+    loader: boolean;
+
+    constructor(private conductService: ConductService, private router: Router) {
+    }
+
+    /**
+     * This method used for register test attendee.
+     */
+    registerTestAttendee() {
+        this.loader = true;
+        this.registrationUrl = window.location.pathname;
+        this.testId = this.registrationUrl.substring(this.registrationUrl.indexOf('/conduct/') + 9, this.registrationUrl.indexOf('/register'));
```

Comment:
I didn't understand this process. What you are doing here? 
this.registrationUrl.substring(this.registrationUrl.indexOf('/conduct/') + 9, this.registrationUrl.indexOf('/register'));